docs/topics/whatsnew/whatsnew2120.md

```diff
@@ -467,8 +467,8 @@ Starting with Kotlin 2.1.20, the standard library provides the ability to repres
 was previously only available in [`kotlinx-datetime`](https://kotlinlang.org/api/kotlinx-datetime/), an official Kotlin
 library.
 
-The [`kotlinx.datetime.Clock`](https://kotlinlang.org/api/kotlinx-datetime/kotlinx-datetime/kotlinx.datetime/-clock/) interface
-is introduced to the standard library as `kotlin.time.Clock` and the [`kotlinx.datetime.Instant`](https://kotlinlang.org/api/kotlinx-datetime/kotlinx-datetime/kotlinx.datetime/-instant/)
+The `kotlinx.datetime.Clock` interface
+is introduced to the standard library as `kotlin.time.Clock` and the `kotlinx.datetime.Instant`
```

Comment:
Should we put references to Clock and Instant provided by the standard library? Like https://kotlinlang.org/api/core/2.1/kotlin-stdlib/kotlin.time/-clock/

Reply:
Yes! In fact, I updated all links to be from the standard library 2.1 version.